client-sdk/ts-web/core/playground/src/index.js

```diff
@@ -96,16 +64,48 @@ const nic = new oasis.client.NodeInternal('http://localhost:42280');
             console.log('sent');
         }
 
+        // Try verifying transaction signatures.
+        {
+            // TODO: Make sure this is the block with the transaction we sent above.
+            const genesis = await nic.consensusGetGenesisDocument();
```

Comment:
You could use the new `GetChainContext` method instead of fetching the whole genesis document.

Reply:
that's a good idea. I'll file that. I also noticed that we have a new GetAccountNonce thing that could replace a part where we get the entire account to read the nonce.

Reply:
#48

Reply:
Yeah the `GetSignerNonce` has been there for a while though.